cli/tests/unit/console_test.ts

```diff
@@ -799,6 +797,12 @@ unitTest(function consoleTestWithObjectFormatSpecifier(): void {
   );
 });
 
+unitTest(function consoleTestWithStyleSpecifier(): void {
+  assertEquals(stringify("%cfoo%cbar"), "%cfoo%cbar");
+  assertEquals(stringify("%cfoo%cbar", ""), "foo%cbar");
+  assertEquals(stripColor(stringify("%cfoo%cbar", "", "color: red")), "foobar");
```

Comment:
I don't think we should strip the color here. We should be testing that the color actually applied correctly.

Reply:
Yeah I intended to only test that the thing is stripped because colours are annoying to test, but it would be pretty simple copy-pastes.

Reply:
Ah, that's right. `NO_COLOR=1` is set for `cargo test js_unit_tests` so we can't. I'll expose `cssToAnsi` for testing instead.